### PR TITLE
feat: Breadcrumbs

### LINKED ISSUE #486: Breadcrumbs
## Description
Add breadcrumbs to navigate in the platform

## Additional context

- Structure: https://docs.google.com/spreadsheets/d/1U6YbJRzXg9XRa3KOMYQwuyGBgoNpP7ntDQ0hbDs5JT8/edit#gid=0
- Example screen:  https://www.figma.com/file/WWQO87EwdsHaHHRBTEexjJ/Display-dataset-R1.3?node-id=1306%3A1058

### PR DESCRIPTION
**Pull request by josebui**
## Description

- Added breadcrumbs

### Checklist
- [x] Corresponding issue has been opened
- [x] New tests added
- [x] English strings reviewed and copyedited
- [x] Strings are localized
- [x] Verified on mobile
- [x] Verified on desktop
- [x] Verified accessibility ([Accessibility - Web applications](https://docs.google.com/document/d/1VwPDyLqw7r_iISMgDZr1FdAmAqqOIIVIxQqsYA-HaHE/edit?usp=sharing))

### Related Issues
Fixes #486 

### Verification steps

image

### CODE REVIEW

File: src/localization/locales/en-US.json [hunk 1]
```diff
@@ -119,7 +119,9 @@
         "members_list_pending_confirmation_message": "Do you want to deny {{userName}}’s request for membership in {{name}}?",
         "members_list_pending_confirmation_button": "Deny Request",
         "members_list_pending_reject": "Deny",
-        "home_pending_message": "Waiting for the group manager’s approval"
+        "home_pending_message": "Waiting for the group manager’s approval",
+        "breadcrumbs_view": "{{groupName}}",
```
Comment (paulschreiber) [inline src/localization/locales/en-US.json:123]: question: if this is just a variable, do we need a localized string for it?

**Review by paulschreiber — CHANGES_REQUESTED** [review]

File: src/localization/locales/en-US.json [hunk 2]
```diff
@@ -191,7 +193,8 @@
         "nav_label": "Main navigation",
         "nav_label_short": "Main",
         "skip_to_main_content": "Skip to main content",
-        "skip_to_main_navigation": "Skip to main navigation"
+        "skip_to_main_navigation": "Skip to main navigation",
+        "breadcrumbs_label": "Breadcrumbs"
```
[the remark this answers is not in the document]
Comment (josebui) [reply src/localization/locales/en-US.json:197]: I based the implementation on this: https://a11y-style-guide.com/style-guide/section-navigation.html#kssref-navigation-breadcrumbs. 

If the question is about the language and what that means from the product side please let Derek and Hana know your concern

File: src/localization/locales/en-US.json (hunk 1 above)
Comment (josebui) [reply src/localization/locales/en-US.json:123]: @paulschreiber This side effect supports the flexibility to define labels for each page that needs to have breadcrumb without having manage a complex structure for each one of the pages

**Review by paulschreiber — APPROVED** [review]
@josebui This looks good. Are you able to use a pathname library for the work in Routes.js?

**Comment by josebui** [comment]
@paulschreiber I should, I missed that change yesterday

**Comment by josebui** [comment]
@paulschreiber I checked and `path-browserify` is not the best solution for URL paths, it is used for server paths, in the case of the other place we are using it makes more sense since it is used to parse a filename

**Pull request merged**